### PR TITLE
update pylorcon2 from python 2.7 to python 3.x

### PR DESCRIPTION
**Pull request by thex1le**
With Python 2.7 going EOL, update this code to work on python 3

### CODE REVIEW

File: pylorcon2/PyLorcon2.c [hunk 1]
```diff
@@ -22,7 +22,7 @@
 */
 
 
-#include <python2.7/Python.h>
+#include <python3.6/Python.h>
```
Comment (Arfrever) [inline pylorcon2/PyLorcon2.c:25]: Please do not hardcode any Python version.
Just use `#include <Python.h>`.
`setup.py` already passes `-I/usr/include/pythonX.Y` for appropriate Python version.

File: pylorcon2/PyLorcon2.h [hunk 2]
```diff
@@ -21,7 +21,7 @@
     Author: Ezequiel Gutesman (gutes)  <egutesman@gmail.com>
 */
 
-#include <python2.7/Python.h>
+#include <python3.6/Python.h>
```
Comment (Arfrever) [inline pylorcon2/PyLorcon2.h:24]: Same here.

**Comment by thex1le** [comment]
pylorcon2 is going away this is not needed

**Comment by Arfrever** [comment]
> pylorcon2 is going away this is not needed

Have you considered existence of other software projects which might use PyLorcon2?
An example is [airdrop-ng](https://github.com/aircrack-ng/aircrack-ng/blob/bf215097579b45bf34f9545f000dd9fe0211d7b0/scripts/airdrop-ng/airdrop-ng#L23) tool from [aircrack-ng](https://github.com/aircrack-ng/aircrack-ng).

What is replacement for PyLorcon2?

**Comment by thex1le** [comment]
The solution is to migrate everything over to
https://github.com/kismetwireless/lorcon/tree/master/pylorcon_ffi
I am the author of airdrop-ng and one of these days I will get around to
it. So much work, so little time. In the meantime I have python3 support
fixed in my branch however airdrop-ng2 is still python 2 based so it's
going to need a lot of work to become compliant. Are you a major user of
that tool?

**Pull request closed without merging**